src/injector/Makefile

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+.PHONY: help
+help: ## Display this help information
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) \
+	  | sort | awk 'BEGIN {FS = ":.*?## "}; \
+	  {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+clean: ## Clean the build directory
+	rm -rf target
+
+build-injector-linux: build-injector-linux-amd build-injector-linux-arm ## Build the Zarf injector for AMD64 and ARM64
+
+build-injector-linux-amd: ## Build the Zarf injector for AMD64
+	rustup target add x86_64-unknown-linux-musl
+
+	if [ "$(shell uname -m)" = "arm64" ] || [ "$(shell uname -m)" = "aarch64" ]; then \
+		test -s x86_64-linux-musl-cross || curl https://musl.cc/x86_64-linux-musl-cross.tgz | tar -xz; \
+		export PATH="$$PWD/x86_64-linux-musl-cross/bin:$$PATH"; \
+		cargo build --target x86_64-unknown-linux-musl --release; \
+		x86_64-linux-musl-strip target/x86_64-unknown-linux-musl/release/zarf-injector; \
+	elif [ "$(shell uname -m)" = "x86_64" ]; then \
+		cargo build --target x86_64-unknown-linux-musl --release; \
+        strip target/x86_64-unknown-linux-musl/release/zarf-injector; \
```

Comment:
strip should not be needed as `cargo.toml` is already configured to do so. https://github.com/johnthagen/min-sized-rust

Reply:
done